src/components/TreeGrid.ts

```diff
@@ -203,6 +223,8 @@ const TreeGridInner = <T extends RowContents>(props: TreeGridPropsInner<T>) => {
       );
     },
     border: false,
+    // Clamp lets us place the first occurrence of the selected concept in the middle of the grid
+    scrollToRow: _.clamp(0, visibleRows.length - 1, initialRow),
```

Comment:
- What does this comment mean? How is clamp placing something in the middle of the grid?
- Is `clamp()` needed? Is it only to handle the case where there are not enough rows showing to center the scrolled row?
- right now I'm not sure if `undefined` is properly supported. It would be nice to write it so that nothing is done if `undefined` is passed in. Something like:
```
  const scrollToRow =
    props.openedConceptId === undefined
      ? undefined
      : _.flow(
          _.findIndex((row: Row<T>) => row.contents.id === props.openedConceptId),
          (index) => (index === -1 ? -1 : _.clamp(0, visibleRows.length - 1, index + _.floor(rowsShowing / 2)))
        )(visibleRows);
...
  return h(Grid, {
    ref: treeGrid,
...
    border: false,
    scrollToRow,
  });
```